examples/search_query.rs

```diff
@@ -0,0 +1,13 @@
+use rspotify::search::SearchQuery;
+
+#[tokio::main]
+async fn main() {
+    let query: String = SearchQuery::default()
+        .any("any")
+        .artist("Lisa")
+        .track("Demon slayer")
+        .album("Another Album")
+        .into();
+
+    println!("{}", query);
```

Comment:
I think it's not a full example, right? The `SearchQuery` should combine with the `search`  endpoint?